code/game/machinery/cryopod.dm

```diff
@@ -432,3 +432,28 @@ GLOBAL_VAR_INIT(cryopods_enabled, FALSE)
 	log_admin(span_notice("[key_name(target)] entered a stasis pod."))
 	message_admins("[key_name_admin(target)] entered a stasis pod. (<A HREF='?_src_=holder;[HrefToken()];adminplayerobservecoodjump=1;X=[x];Y=[y];Z=[z]'>JMP</a>)")
 	add_fingerprint(target)
+
+/obj/machinery/cryopod/JoinPlayerHere(mob/M, buckle)
+	. = ..()
+	open_machine()
```

Comment:
Should probably have it open after the 6 second sleep ends instead of instantly, looks weird when it opens while you're asleep.

Reply:
> Should probably have it open after the 6 second sleep ends instead of instantly, looks weird when it opens while you're asleep.

I don't want to hog them up for 6 seconds to do this. Since this is both "join and leave the game" I wanted the entering to be relatively quick in the cryopod itself.